### PR TITLE
Ci/fix tidy

### LINKED ISSUE #1627: Some issues with the new CI lint?
```The command "git reset --soft ${TRAVIS_COMMIT_RANGE%...*} && git lint" exited with 1.```

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR fixes linting of html-files on travis. Resolves #1627. Removes dependency on tidy since it is a bit tricky to install on travis 

**How to test**:
1. Open Travis PR-check
1. open the "Lint Free" job

**Expected outcome**:
- [x] The linter should OK the delivery-report.html without suggesting to install tidy.
- [x] Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by @dnil
- [x] tests executed by @patrikgrenfeldt

### CODE REVIEW

**Comment by patrikgrenfeldt** [comment]
Test result:
image
👍

**Comment by patrikgrenfeldt** [comment]
I also took the freedom to try to clarify the different jobs by updating their names:
image

File: .travis.yml [hunk 1]
```diff
@@ -41,12 +22,31 @@ jobs:
...
+              - pip install . -r requirements-dev.txt
+          script: pip check
+
+        - name: "PEP8 Compliance"
           if: type = pull_request
           install: pip install black
           script: git --no-pager diff --name-only --diff-filter=AM $TRAVIS_COMMIT_RANGE | grep -F ".py" | xargs black --check
 
-        - name: "Pylint score"
+        - name: "Code Quality Score"
```
Comment (dnil) [inline .travis.yml]: Hm, I kind of liked pylint score better. Judging from the lints its a lot about whitespace around commas and assignment operators. Its nice and tidy to have that look the same everywhere, but does not affect function in any way.

File: .travis.yml [hunk 2]
```diff
@@ -41,12 +22,31 @@ jobs:
...
+          script: pip check
+
+        - name: "Development Dependencies"
+          if: type = pull_request
+          install:
+              - pip install cython
+              - pip install . -r requirements-dev.txt
+          script: pip check
+
+        - name: "PEP8 Compliance"
```
Comment (dnil) [inline .travis.yml]: :+1:

File: .travis.yml (hunk 1 above)
Comment (patrikgrenfeldt) [reply .travis.yml]: ok, changed to Pylint Rating

Comment (patrikgrenfeldt) [reply .travis.yml]: We'll change it later if needed..

**Pull request merged**